its/projects/MultiLanguageSupportNonSdk/MultiLanguageSupportNonSdk.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
We could have a single solution with both projects.

Reply:
Compiling a single solution will change the projectBaseDir from the .csproj location to the .sln one. This will require additional nesting to cover the Outside.js and Outside.sql cases.